README.md

```diff
@@ -1,3 +1,66 @@
 [![Build Status](https://api.travis-ci.org/alphagov/paas-cf.svg)](https://travis-ci.org/alphagov/paas-cf)
 
 # paas-cf
+
+This repository contains [Concourse](http://concourse.ci/) pipelines and related [Terraform](https://terraform.io/) and [BOSH](https://bosh.io/) manifests that allow provisioning of [CloudFoundy](https://www.cloudfoundry.org/) on AWS.
+
+## Getting started
+
+You need the following prerequisite before you will be able to deploy first pipeline.
+
+* Virtualbox
+* Vagrant
+* AWS Access
+
+Provide AWS access keys as environment variables, plus the corresponding terraform variables.
+```
+export AWS_ACCESS_KEY_ID=XXXXXXXXXX
+export AWS_SECRET_ACCESS_KEY=YYYYYYYYYY
+export TF_VAR_AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
+export TF_VAR_AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+```
+
+## Usage
+
+### Prepare the environment
+
+1. Clone repo and boot the vagrant node with concourse:
+
+    ```
+git clone https://github.com/alphagov/paas-cf.git
+cd paas-cf
+vagrant up
+```
+
+2. Install your `fly` binary:
+
+    ```
+sudo curl http://192.168.100.4:8080/api/v1/cli?arch=amd64&platform=$(uname | tr '[:upper:]' '[:lower:]') -o /usr/local/bin/fly && \
+sudo chmod +x /usr/local/bin/fly
+```
+
+3. Login on the local concourse:
+
+     ```
+fly login --concourse-url http://192.168.100.4:8080 sync
+```
+
+### Deploy the deployer concourse
+
+```
+./concourse/scripts/destroy-deployer.sh
```

Comment:
The title on L48 says "deploy" but the command on L51 says "destroy" and is the same command as the section below it (which is about destroying).